boards/nordic/nrf7002dk/board.cmake

```diff
@@ -1,15 +1,8 @@
 # Copyright (c) 2024 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
-if(CONFIG_BOARD_NRF7002DK_NRF5340_CPUAPP_NS OR
-   CONFIG_BOARD_NRF7002DK_NRF5340_CPUAPP_NRF7001_NS)
-  set(TFM_PUBLIC_KEY_FORMAT "full")
```

Comment:
Also the following is only related to non-secure targets so it's unneeded upstream: https://github.com/zephyrproject-rtos/zephyr/blob/733225e4a0322575006cca97b0d1d65f0664843c/boards/nordic/nrf7002dk/board.cmake#L9-L11